indico/web/flask/session.py

```diff
@@ -25,6 +26,11 @@
 from indico.web.util import get_request_user
 
 
+RE_SKIP_REFRESH_SESSION_ENDPOINTS = re.compile(
+    r'core\.session_expiration|assets\.|plugin_.*\.static$'
```

Comment:
I would move `core.session_expiration` out of the regex and check `request.endpoint == 'core\.session_expiration' or ....match()` below. IMHO it's more readable that way because it keeps the assets and non-assets stuff separated.